compute_tools/src/compute.rs

```diff
@@ -338,9 +338,13 @@ impl ComputeNode {
         let lsn = match spec.mode {
             ComputeMode::Primary => {
                 info!("starting safekeepers syncing");
-                let lsn = self
-                    .sync_safekeepers(pspec.storage_auth_token.clone())
-                    .with_context(|| "failed to sync safekeepers")?;
+                let lsn = if let Some(synced_lsn) = spec.skip_sync_safekeepers {
```

Comment:
> we jump to step 1 with the same spec because spec is stored in the k8s resource

This not exactly correct, as now computes fetch spec from the control-plane, but it's cached either way, yes

Reply:
> 1. spec.skip_sync_safekeepers is set to some incorrect LSN

This would be a bug though. Let's not set it to an incorrect LSN